src/methods/vault/requests/getExitQueuePositions/index.ts

```diff
@@ -21,6 +22,8 @@ const getExitQueuePositions = async (input: GetExitQueuePositionsInput) => {
 
   validateArgs.address({ vaultAddress, userAddress })
 
+  const duration = await fetchExitRequestDuration({ options, vaultAddress, userAddress })
```

Comment:
Please remove async and await, otherwise, it will not be the abort promise with abort() method

Try to use AbortPromise.all() instead

Reply:
https://github.com/stakewise/v3-sdk/pull/90/commits/5b426a7e1634f48071aaad27e6edfe7c5f5e1947